avm/res/api-management/service/README.md

```diff
@@ -1041,7 +1041,7 @@ The diagnostic settings of the service.
 | [`logAnalyticsDestinationType`](#parameter-diagnosticsettingsloganalyticsdestinationtype) | string | A string indicating whether the export to Log Analytics should use the default destination type, i.e. AzureDiagnostics, or use a destination type. |
 | [`logCategoriesAndGroups`](#parameter-diagnosticsettingslogcategoriesandgroups) | array | The name of logs that will be streamed. "allLogs" includes all possible logs for the resource. Set to '' to disable log collection. |
 | [`marketplacePartnerResourceId`](#parameter-diagnosticsettingsmarketplacepartnerresourceid) | string | The full ARM resource ID of the Marketplace resource to which you would like to send Diagnostic Logs. |
-| [`metricCategories`](#parameter-diagnosticsettingsmetriccategories) | array | The name of logs that will be streamed. "allLogs" includes all possible logs for the resource. Set to '' to disable log collection. |
+| [`metricCategories`](#parameter-diagnosticsettingsmetriccategories) | array | The name of logs that will be streamed. "allMetrics" includes all possible logs for the resource. Set to '' to disable log collection. |
```

Comment:
Hey @AlexanderSehr are all these `allLogs` -> `allMetrics` changes intended? I've noticed they've been added after previous review

Reply:
Hey Erika, they are. They popped up when running all tests again

Reply:
They are, but his PR would fix them (@ChrisSidebotham)

Reply:
@eriqua, how's it going? ;)

Reply:
Still has conflicts and changes are added any time I get to review it :dagger:

Let me know once ready and please let's freeze it after that. 
If additional changes are required please let me know since understanding it from commits is just making it worse.

Reply:
I would very much appreciate that. The PR keeps on groing as it's open for 3 weeks and is the only one that allows bugfixes across files without causing conflicts - because it's the on PR with manual fixes (as opposed to e.g. the telemetry PR). ⚠️ 
It was only 'not' ready again because of the merge of #760 ;) Anyhow - just resolved the conflict and should be ready to go again @eriqua. (: